profiles/kentik_snmp/raritan/dominion.yml

```diff
@@ -0,0 +1,66 @@
+# https://mibs.observium.org/mib/RemoteKVMDevice-MIB/
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.3.2.10    # Raritan Dominion SXII DSX2-32
+  - 1.3.6.1.4.1.13742.*    # Raritan Devices
+
+provider: kentik-switch
+
+metrics:
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: remoteKVMDeviceGet
+      OID: 1.3.6.1.4.1.13742.3.1
+    symbols:
+      # Percent current memory usage
+      - name: systemUsageMemory
+        OID: 1.3.6.1.4.1.13742.3.1.1
```

Comment:
Per the walk this need to be OID: 1.3.6.1.4.1.13742.3.1.1.0
Usually Scalar OID's will need that but mib documentation usually doesn't include it

Reply:
Changed to scalar and added the 0
  - MIB: RemoteKVMDevice-MIB
    symbols:
      # Percent current memory usage
      - name: systemUsageMemory
        OID: 1.3.6.1.4.1.13742.3.1.1.0
        tag: MemoryUtilization
        poll_time_sec: 60